sdk/storage/storage-queue/test/node/messageidclient.spec.ts

```diff
@@ -194,7 +195,14 @@ describe("MessageIdClient Node.js only", () => {
           expiresOnTimestamp: 12345
         })
     };
-    const newClient = new MessageIdClient("https://queue", tokenCredential);
+    const newClient = new MessageIdClient(
+      extractConnectionStringParts(getConnectionStringFromEnvironment()).url +
+        "/" +
+        queueName +
+        "/messages/" +
+        "fake-message-id",
```

Comment:
I feel this is overkill and adds indirection to people reading the code.  Could we provide a fake url that satisfies the connection string parsing?

Reply:
My point is I'd like be able to look at the test and quickly understand what it is testing.

---
In reply to: [331348593](https://github.com/Azure/azure-sdk-for-js/pull/5388#discussion_r331348593) [](ancestors = 331348593)

Reply:
So, you'd prefer `https://${accountName}.queue.core.windows.net/queueName` over `extractConnectionStringParts(getConnectionStringFromEnvironment()).url` ?

Reply:
Any fake url that is testing the parsing logic? since we don't really interact with Azure service I believe.

Reply:
Yeah, I'll update